trymito.io/components/Header/Header.tsx

```diff
@@ -66,6 +66,27 @@ const Header = (): JSX.Element => {
                         </div>
                       </Link>
                     </li>
+
+                    <li>
+                      <Link href='/infrastructure-integration-python-tool'>
+                        <div className={classNames(headerStyles.dropdown_item)}>
+                          <div className={classNames(headerStyles.dropdown_item_row)}>
+                            <Image src={'/step-icons/SettingsIconThin.svg'} alt='Customize Mito' width={20} height={20} />
+                            <p className='margin-top-0'>
+                              Custom Functionality
+                            </p>
+                          </div>
+                          <div className={classNames(headerStyles.dropdown_item_row)}>
+                            <p className={headerStyles.dropdown_item_subtext}>
+                              BYO spreadsheet functions and imports methods.
```

Comment:
```suggestion
                              Bring-your-own spreadsheet formulas and imports methods.
```

Formulas is more clear (despite not being exactly our language) perhaps.